src/styles/Header.scss

```diff
@@ -0,0 +1,40 @@
+.header{
+    background-color: $darkblue;
+    height: 100px;
+    display: flex;
+    align-items: center;
+    &__items{
+        display: flex;
+        justify-content: space-between;
+        &__text{
+            color: $white;
+            font-size: 40px;
+            & span{
+                font-weight: bold;
+                color: $lightblue
+            }
+        }
+        &__links{
+            display: flex;
+            width: 400px;
+            justify-content: flex-end;
+            align-items: center;
+            & .link{
+                font-size: 20px;
+                text-decoration: none;
+                color: $white;
+                margin-left: 30px;
```

Comment:
zou opteren om in __links gebruik te maken van gap:30px -> oogt wat mooier in de inspector